config/config-example.json

```diff
@@ -31,6 +31,8 @@
     "token": ""
   },
   "events": [
+    "message"
```

Comment:
Each of these needs a comma at the end, and preferably have them after `guildMemberUpdate` and `messageReactionRemove`, respectively. Because alphabet.

I'd do it myself, but the whole "Approve with changes" option seems to have disappeared for me.

Reply:
Alright, I recommited with those changes